### PR TITLE
backend: Refactor backend/pkg/helm/release.go

### PR DESCRIPTION
**Pull request by HariCharan-001**
Refactored code in [headlamp/backend/pkg/helm/release.go](https://github.com/headlamp-k8s/headlamp/blob/main/backend/pkg/helm/release.go#L628) file.

backend: Refactor release.go UpgradeRelease to be shorter #1793
_Abstracted out repetitive parts, made the code shorter and more-readable._

 backend: Refactor, Add returnResponse method for HTTP responses
_Abstracted out repetitive parts, made the code shorter and more-readable._

### CODE REVIEW

**Review by illume — COMMENTED** [review]
Nice one. Thank you.

I left a question for your consideration...

File: backend/pkg/utils/utils.go [hunk 1]
```diff
@@ -10,3 +16,8 @@ func Contains[T comparable](elems []T, v T) bool {
 
 	return false
 }
+
+func HandleError(w http.ResponseWriter, releaseName string, err error, message string, status int) {
```
[the remark this answers is not in the document]
Comment (HariCharan-001) [reply backend/pkg/utils/utils.go]: Moved to release.go

**Comment by HariCharan-001** [comment]
Updated. 👍🏻

**Review by illume — APPROVED** [review]
🎉🎈

Thanks.

**Pull request merged**